Objects/listobject.c

```diff
@@ -2145,12 +2145,21 @@ list.sort
     key as keyfunc: object = None
     reverse: bool(accept={int}) = False
 
-Stable sort *IN PLACE*.
+Sort the list in ascending order.
```

Comment:
Specifiy the return value.  Add 'and return None'.
See next comment.

Reply:
Agree.